lib/licensee/project_files/license_file.rb

```diff
@@ -29,6 +29,12 @@ class LicenseFile < Licensee::Project::File
         //                                           => 0.0  # Catch all
       }.freeze
 
+      # CC-NC and CC-ND are not open source licenses and should always be
+      # detected as the "other" license
```

Comment:
Clarified via 3579d6f. I was wrong. In Licensee, the existence of a LicenseFile shows that there is a license. If no matcher returns a match (e.g., DiceMatcher returns nil for match), that means the license hasn't been detected (but that there is a license file). In the GitHub API, we go a step further and collapse that into `other`, nil, or the detected license.